tao_compiler/mlir/disc/transforms/tests/disc-fold-quantize.mlir

```diff
@@ -44,6 +44,49 @@ func.func @quantize_per_tensor_asymmetric_uint8() ->  tensor<2x3xui8> {
   return %y : tensor<2x3xui8>
 }
 
+func.func @quantize_per_channel_symmetric_int8_fp16() ->  tensor<2x3xi8> {
+  // CHECK{LITERAL}: mhlo.constant dense<[[10, 20, 30], [20, 25, 30]]> : tensor<2x3xi8>
+  // CHECK-NOT: mhlo_disc.quantize
+  %x = mhlo.constant dense<[[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]> : tensor<2x3xf16>
```

Comment:
Do we need some method to specify which channel to quant? input or outpu?

Reply:
the `axis` attr of `mhlo_disc.quantize` is used to specify the channel to quant:)